src/viam/api/buf.lock

```diff
@@ -4,11 +4,11 @@ deps:
   - remote: buf.build
     owner: googleapis
     repository: googleapis
-    commit: 28151c0d0a1641bf938a7672c500e01d
+    commit: a86849a25cc04f4dbe9b15ddddfbc488
```

Comment:
Do we still need these version pins? They were added as part of RSDK-5441. Unfortunately the ticket description lacks sufficient detail to understand what motivated the pin in the first place.

Reply:
That's a good question. My intuition is no, we don't need to pin here but I'm honestly not positive. @cheukt do you know?

That said, I have a slight preference for leaving as-is for now, just for the sake of minimizing the footprint of this PR (we already have changes to motion APIs). Not a strong preference though, happy to revisit!